web/src/app/schedules/calendar/ScheduleCalendar.tsx

```diff
@@ -123,81 +182,90 @@ function ScheduleCalendar(props) {
     )
   }
 
-  const getCalEvents = (shifts, _tempScheds, userOverrides) => {
-    const tempSchedules = _tempScheds.map((sched) => ({
-      start: sched.start,
-      end: sched.end,
+  const getCalEvents = (
```

Comment:
From what I can tell, an event is "Fixed' if it's a temp schedule event or temp schedule shift event. I just removed the terminology of "fixed" and made it more clear, but the term "fixed" may be being used in react-big-calendar. I couldn't find any docs mentioning it though so I'm assuming that's a term we're using outside of react-big-calendar.